docs/UserGuide.md

```diff
@@ -627,9 +627,8 @@ This can be useful when you [start a lesson](#starting-a-lesson-startlesson-c) b
 <box type="info">
 
 **Information:**
-- NUMBER_OF_LESSONS takes numbers such as `0`,`1`,`2`,`3`
+- NUMBER_OF_LESSONS must be a valid non-negative integer such as `0`,`1`,`2`,`3`
```

Comment:
Just nitpicking... perhaps you could say "positive" integer instead

Reply:
Mine can accept 0 ahahah

Reply:
Mine can accept 0, that's why I did not follow the rest